src/EA.Iws.Web/Views/Applicant/Home.cshtml

```diff
@@ -43,4 +44,66 @@
 
 <h2 class="heading-medium">@HomeResources.HeaderManageYourNotifications</h2>
 
-@Html.Action("_GetUserNotifications")
\ No newline at end of file
+@if (ViewBag.Errors != null && ((IEnumerable<string>)ViewBag.Errors).Any())
+{
+    <p>@HomeResources.ErrorMessage</p>
+}
+else if (Model.Count == 0)
+{
+    <p>@HomeResources.MessageNoNotification</p>
+}
+else
+{
+    <table class="odd-table" title=@HomeResources.TitleManageNotifications>
```

Comment:
Whenever you have a HTML attribute and the value is from a variable (like a resource file), it still needs to be wrapped in double quotes, so it should be `title="@HomeResources.TitleManageNotifications"`.